CodeEdit/Features/UtilityArea/Views/UtilityAreaView.swift

```diff
@@ -39,8 +41,17 @@ struct UtilityAreaView: View {
                         model.isMaximized.toggle()
                     } label: {
                         Image(systemName: "arrowtriangle.up.square")
+                            .foregroundColor(
+                                model.isMaximized
+                                ? Color(.controlAccentColor)
+                                : model.selectedTerminals.isEmpty ? Color(.secondaryLabelColor) : .gray
+                            )
+                            .frame(width: 24, height: 24, alignment: .center)
+                            .contentShape(Rectangle())
+                            .opacity(controlActiveState == .inactive ? 0.5 : 1)
+                            .symbolVariant(model.isMaximized ? .fill : .none)
                     }
-                    .buttonStyle(.icon(isActive: model.isMaximized, size: 24))
+                    .buttonStyle(HighlightPressButtonStyle())
                 }
             }
```

Comment:
```suggestion
                    .buttonStyle(.icon(isActive: model.isMaximized, size: 24))
                }
            }
            .colorScheme(
                model.selectedTerminals.isEmpty
                ? colorScheme
                : matchAppearance && darkAppearance
                ? themeModel.selectedDarkTheme?.appearance == .dark ? .dark : .light
                : themeModel.selectedTheme?.appearance == .dark ? .dark : .light
            )
```

Reply:
Note: We are taking this from [here](https://github.com/CodeEditApp/CodeEdit/blob/main/CodeEdit/Features/UtilityArea/TerminalUtility/UtilityAreaTerminalView.swift#L183-L189). Because the maximize button is overlaid, this `colorScheme` doesn't get applied, so we need to apply it here as well.